plugins/services/src/js/service-configuration/ServiceConfigDisplay.js

```diff
@@ -38,9 +38,9 @@ const DEFAULT_DISPLAY_COMPONENTS = [
       ServicePlacementConstraintsConfigSection,
       ServiceNetworkingConfigSection,
       ServiceStorageConfigSection,
+      ServiceHealthChecksConfigSection,
```

Comment:
🤔 Why are these reordered?

Reply:
Not a big deal, but looks like it was just accidentally committed.

Reply:
Before Environment config was showing before Health Check config. I switched the order of the two to keep consistent with how we do it within the configuration flow.

Reply:
Ok sounds good 👍